projboard/views.py

```diff
@@ -16,7 +16,7 @@ def create_article(request):
         if form.is_valid():
             form.save()
             # TODO when we end to create the "my articles" page, change the redirect
-            return redirect('board')
+            return render(request, 'projboard/board.html', {})
     else:
         form = CreateArticleForm(initial={'user_id': user})
```

Comment:
I am apologies but I did not understand  for what you meant..

Reply:
I think @Yarboa means that you should add if request.method == "GET" ,
because in  delete case the func should act defirinte ,
in this case above poth delet and get wull act similar .

Reply:
It seems this  method will create on HTTP DELETE will behave like HTTP GET isnt it?

Reply:
But my action in the form is "POST" , why should I handle HTTP DELETE ?
I don't think the user can send a request from type DELETE 

WDYT?

Reply:
Correct, 
But many HTTP requests will trigger this logic

You should care, and you should also propose tests in the client for that
Create article build from 2:
1 HTTP GET to retrieve the page
2 HTTP POST to send data to backend

* But HTTP DELETE will return the page also, you can return
message or 404 response, or redirect somewhere 

In that case you handle the POST

You should care, tests and review reduce some risks for code misbehavior

You should add more tests here

Reply:
Done at [98a2d9a](https://github.com/redhat-beyond/Smarticle/pull/70/commits/98a2d9af404f43b381df51438456b221a003e754)